### PR TITLE
PD-116 Update Core to Scale Migration Article

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates the MigratingFromCORE.md article content by adding a section on preparing for migration, adding Enterprise content, adds content on when the CORE update function works to use it to migrate to SCALE. And it makes other style/organizational improvements. When failover works properly, this article will include some information on migrating a non-Enterprise (HA) system from CORE to SCALE. This content is not included yet.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 1]
```diff
@@ -11,34 +11,76 @@ tags:
...
+{{< /hint >}}
 
 TrueNAS SCALE is Linux based, so it does not support FreeBSD GELI encryption.
 If you have GELI-encrypted pools on your system that you plan to import into SCALE, you must migrate your data from the GELI pool to a non-GELI encrypted pool *before* migrating to SCALE.
 
 TrueNAS SCALE validates the system certificates when a CORE system migrates to SCALE. When a malformed certificate is found, SCALE generates a new self-signed certificate to ensure system accessibility.
 
-## Migration Methods
+{{< hint warning >}}
+Plugins to not migrate from CORE to SCALE. Save the configuration information for your plugin and back up stored data.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:31]:
```suggestion
Plugins do not migrate from CORE to SCALE. Save the configuration information for your plugin and back up stored data. 
```

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 2]
```diff
@@ -11,34 +11,76 @@ tags:
...
 TrueNAS SCALE is Linux based, so it does not support FreeBSD GELI encryption.
 If you have GELI-encrypted pools on your system that you plan to import into SCALE, you must migrate your data from the GELI pool to a non-GELI encrypted pool *before* migrating to SCALE.
 
 TrueNAS SCALE validates the system certificates when a CORE system migrates to SCALE. When a malformed certificate is found, SCALE generates a new self-signed certificate to ensure system accessibility.
 
-## Migration Methods
+{{< hint warning >}}
+Plugins to not migrate from CORE to SCALE. Save the configuration information for your plugin and back up stored data. 
+After completing the SCALE install, add the application using the **Apps** option. 
+If your CORE plugin is not listed as an available application in SCALE, use the **Launch Docker Image** option to add it as and application and import data from the backup into a new SCALE dataset for the application.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:33]:
```suggestion
If your CORE plugin is not listed as an available application in SCALE, use the **Launch Docker Image** option to add it as an application and import data from the backup into a new SCALE dataset for the application.
```

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 3]
```diff
@@ -11,34 +11,76 @@ tags:
...
+{{< /hint >}}
+
+### Migration Methods
+You can migrate from CORE to SCALE through an upgrade or clean install using and <kbd>iso</kbd> file, or in some CORE 13.0 release you can migrate using the CORE UI Upgrade function with the SCALE update file downloaded from the website.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:37]:
```suggestion
You can migrate from CORE to SCALE through an upgrade or clean install using an <kbd>iso</kbd> file, or in some CORE 13.0 release you can migrate using the CORE UI Upgrade function with the SCALE update file downloaded from the website. 
```

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 4]
```diff
@@ -11,34 +11,76 @@ tags:
...
+{{< /hint >}}
+Before you attempt to migrate your CORE system to SCALE:
+
+1. Upgrade your CORE system to the most recent publicly-available CORE version. 
+   TrueNAS systems on 12.0x or lower should update to the latest CORE 13.0 release (e.g 13.0-U2 or U4 when released) prior to migrating to SCALE. 
+   CORE systems at release 13.0-Ux can use the [iso upgrade](#migrating-using-an-iso-file-to-upgrade) method to migrate to SCALE. 
+   Lower releases of CORE (12.0-Ux) must do a clean install with the SCALE <kbd>iso</kbd> file.
+2. Verify the root user is not locked. 
+   Go to **Accounts > Users**, use **Edit** for the root user to view current settings and confirm **Lock User** is not selected.
+3. After updating to the latest publicly available release of CORE, download your system configuration file and a debug file.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:55]:
```suggestion
3. After updating to the latest publicly-available release of CORE, download your system configuration file and a debug file. 
```

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 5]
```diff
@@ -11,34 +11,76 @@ tags:
...
+   Go to **Accounts > Users**, use **Edit** for the root user to view current settings and confirm **Lock User** is not selected.
+3. After updating to the latest publicly available release of CORE, download your system configuration file and a debug file. 
+   Keep these files in a safe place in case you need to revert back to CORE with a clean install of the CORE <kbd>iso</kbd> file.
+4. Back up your stored data files. 
+   If you need to do a clean install with the SCALE <kbd>iso</kbd> file, you can import your data into SCALE.
+5. Write down your network configuration information to use if you do a clean install of SCALE from an <kbd>iso</kbd> file.
+   {{< include file="/_includes/NetworkInstallRequirementsSCALE.md" type="page" >}}
+
+{{< hint warning >}}
+Plugins to not migrate to SCALE. Save plugin (or jails) configuration information and back up stored data.
```
Comment (clairewrtr) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:63]:
```suggestion
Plugins do not migrate to SCALE. Save plugin (or jails) configuration information and back up stored data. 
```

**Review by clairewrtr — COMMENTED** [review]
Looks good, minor changes.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**